hist/histpainter/src/TGraphPainter.cxx

```diff
@@ -3320,8 +3320,8 @@ void TGraphPainter::PaintGraphReverse(TGraph *theGraph, Option_t *option)
    Double_t YA2 = theGraph->GetYaxis()->GetXmax();
    Double_t dX  = XA1+XA2;
    Double_t dY  = YA1+YA2;
-   Double_t newX[N];
-   Double_t newY[N];
+   Double_t *newX = new Double_t[N];
+   Double_t *newY = new Double_t[N];
```

Comment:
How about `std::vector<Double_t> newX(N);`?

Reply:
Well, this was the most straightforward change, but let's ask the code owner @couet